airbyte-integrations/connectors/source-s3/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: 69589781-7828-43c5-9f63-8925b1c1ccc2
-  dockerImageTag: 4.9.0
+  dockerImageTag: 5.0.0-rc.1
```

Comment:
Beware that as or fight now that would trigger a breaking change, which would mean that people would need to opt into it.

This should not stop us, but I wonder if we can clarify whether the platform would treat a major connector version as breaking by default, and if it will not — perhaps we can change our metadata validation as well to not require major version to come with releases.breakingChanges entry. /cc @alafanechere @pedroslopez

Reply:
We don't strictly need to be a 'v5'. But in my head at least the benefits are: not supporting 'v3' state anymore, plus giving a narrative around "migrate to v4 before migrating to v5" seems a very clear and easy-to-communicate message to deliver for users.